resources/puppetlabs/lein-ezbake/template/global/ext/fpm.rb

```diff
@@ -189,25 +190,28 @@
     options.systemd_el = 1
   elsif options.operating_system == :el && options.os_version >= 7 # systemd el
     if ! options.is_pe
+      # https://bugzilla.redhat.com/show_bug.cgi?id=2224427
       fpm_opts << "--depends tzdata-java"
-      options.java =
-        case options.platform_version
-        when 8
-          # rpm on Redhat 7 may not support OR dependencies
-          if options.os_version == 7
-            'java-11-openjdk-headless'
-          elsif options.os_version == 8
-            '(java-17-openjdk-headless or java-11-openjdk-headless)'
-          elsif options.os_version >= 9
-            'java-17-openjdk-headless'
-          else
-            fail "Unrecognized el os version #{options.os_version}"
-          end
-        when 7
-          'java-11-openjdk-headless'
+      case options.platform_version
+      when 8
+        # rpm on Redhat 7 may not support OR dependencies
+        if options.os_version == 7
```

Comment:
Do we still need EL7 specific code?

Reply:
We're still building el7 for the moment.  But I'd be in favor of dropping it if everyone else is.

Reply:
@ekohl I think you rely on puppet 7 for satellite, but do you also need it for EL7?

Reply:
EL7 isn't entirely dead downstream but that's more on the agent side. I think we can stop building server because at least in Satellite we don't support that.